### ISSUE TITLE
Error AuthenticationRequiredError (401): X-Apikey header is missing

### ISSUE THREAD

**Issue by RedFurrFox** [Dextroz/virustotal-python#35]
I am trying to integrate your module to our guilded bot using this piece of code:
```@bot.command(name="urlscan", aliases=["scan", "check", "inspect"])
async def scan(ctx, *, url):
  print(f"{space.a}{pointer.c} Scan command triggered.")
  print(f'{space.b}{pointer.d} Data: "{url}"')
  with virustotal_python.Virustotal(VirToken) as vtotal:
    try:
        resp = vtotal.request("urls", data={"url": url}, method="POST")
        # Safe encode URL in base64 format
        # https://developers.virustotal.com/reference/url
        url_id = urlsafe_b64encode(url.encode()).decode().strip("=")
        report = vtotal.request(f"urls/{url_id}")
        await ctx.send(embed=guilded.Embed(
    title="Scan Command",
    description=f"""Scan info for {url}\n\nObjectType:\n```{report.object_type}```\nData:\n```{report.data}```""",
    color=embed_color))
    except virustotal_python.VirustotalError as err:
        await ctx.send(embed=guilded.Embed(
    title="Scan Command",
    description=f"Failed to send URL: {url} for analysis and get the report: {err}",
    color=embed_color))
```

Then whenever I run this, I always receive this kind of error:
![image](https://user-images.githubusercontent.com/80197308/204266798-eeb20e72-e22f-4568-b6bc-b00b1941781a.png)

I wish someone would help me with this issue. Thanks!

**Comment by dbrennand** [comment]
Hi @RedFurrFox 

Thanks for checking out the library. 

Are you using the latest version available `1.0.0`:

```bash
pip list | grep virustotal-python
```

I can see you've taken the example from the README, I just tested with `1.0.0` and all is working as expected for me. HTTP status `401` indicates that you may not have provided the API key correctly, can you please double check that the `VirToken` variable is definitely populated with your API key?

<html>
<body>


401 | AuthenticationRequiredError | The operation requires an authenticated user. Verify that you have provided your API key.
-- | -- | --



</body>
</html>

Taken from: https://developers.virustotal.com/reference/errors

**Comment by RedFurrFox** [comment]
ohh my bad I forgot to recall that string xd. Thanks for the tip tho
